### ISSUE TITLE
If the wallet is locked we're getting an error message

### ISSUE THREAD

**Issue by kazazor** [rsksmart/rLogin#228]
Whenever the wallet is locked we're showing an error message which is confusing, while actually metamask shows a notification to connect.

![Screen Shot 2022-03-11 at 1 07 27](https://user-images.githubusercontent.com/1577655/157770193-86bc2d9a-ba29-40ee-a903-f41faf6c13c8.png)

Look at what Uniswap are doing in the same case:
![Screen Shot 2022-03-11 at 1 08 47](https://user-images.githubusercontent.com/1577655/157770359-f5061b73-ae9f-46ff-9830-e3fc4e00f147.png)

It detects that the user's wallet is locked and shows an appropriate message.
We should identify it as well and show an appropriate message.

Me and @ilanolkies looked at it together.

**Comment by ilanolkies** [comment]
Please provide the steps to reproduce this one.

We can reproduce this that sounds like your error:
1. Login to the app with Metamask
2. When Metamask pops up, don't accpet/deny
3. Reload the screen (or not, just leave the Metamask request pending)
4. Try to login again
5. New request will fail because there is one already pending in Metamask

See this video

https://user-images.githubusercontent.com/36084092/157880783-2ab1d7c9-4c15-456b-ad79-a1b1bf34b15a.mov

We don't thinks there is a clear way to solve this because it resides on Metamask to be rejecting that second request. What we can do, but we think is on another ticket, is to work on a better feedback when any request is pending to be approved.

**Comment by kazazor** [comment]
It can be reproduced in a simpler way, please try it on https://app.blindex.io

1. Make sure metamask is locked
2. Click on connecting to it in rLogin
3. See that the metamask notification pops
4. Nothing happens on rlogin, so click it again
5. get the error message

In your video is seems Metamask was not locked, which is the core problem I logged in this issue.

https://user-images.githubusercontent.com/1577655/157926107-eb7d673d-2286-4ccc-a0e1-4abb047ef2e2.mov

**Comment by ilanolkies** [comment]
Great! So the thing is when clicking twice. If we address #232 not allowing the user to click twice then this will not be a UX error anymore

**Comment by kazazor** [comment]
If you just don't allow it to click twice, you would still get a weird experience as nothing changed to the user once he clicked on Metamask.. It will seem stuck and still seem as an error to the user.

A better experience would be something like what Uniswap did. WDYT?

**Comment by ilanolkies** [comment]
Yes, sorry. I was unclear. The idea is to show something like "Please confirm in your wallet" instead of showing the landing with all the providers
